src/carts/MagentoCartClient.js

```diff
@@ -188,6 +193,18 @@ class MagentoCartClient extends MagentoClientBase {
         });
     }
 
+    /**
+     * Creates an order from a cart.
+     */
+    order() {
+        return this.withEndpoint(`order`)._execute('PUT')
```

Comment:
You create `MagentoCartClient(args, mapper._mapOrder, 'order')` already with `order` endpoint, is "withEndpoint" still needed?

Reply:
Do we have ticket for that constructor cleanup? If not please create one.